tests/phpunit/CRM/Contribute/Form/Task/StatusTest.php

```diff
@@ -25,12 +25,65 @@ public function tearDown() {
   }
 
   /**
-   * Test update pending contribution
+   * Test update pending contribution with sending a confirmation mail.
    */
-  public function testUpdatePendingContribution() {
+  public function testUpdatePendingContributionWithSendingEmail() {
     $this->_individualId = $this->individualCreate();
     $form = new CRM_Contribute_Form_Task_Status();
 
+    $mut = new CiviMailUtils($this, TRUE);
+    $mut->clearMessages();
+    $mut->assertMailLogEmpty();
```

Comment:
Lines 36 and 85 don't hurt but don't really do anything. It would just be testing the mail utils itself not the feature being added. Sorry that might have come from the first version of the chat post where I made a typo.